pom.xml

```diff
@@ -26,9 +26,8 @@
 
         <maven.surefire.version>3.3.1</maven.surefire.version>
         <maven.checkstyle.version>3.2.1</maven.checkstyle.version>
+        <log4j.version>2.23.1</log4j.version>
```

Comment:
I would group it as it was -> maven versions together, versions for logging stuff together etc.